content/concept-transaction-cost.md

```diff
@@ -19,7 +19,8 @@ Some transactions have different transaction costs:
 |-----------------------|--------------------------|
 | [Reference Transaction](#reference-transaction-cost) (Most transactions) | 10 drops |
 | [Key Reset Transaction](#key-reset-transaction) | 0 |
-| [Multi-signed transaction](reference-transaction-format.html#multi-signing) | 10 drops × (1 + Number of Signatures Provided) |
+| [Multi-signed Transaction](reference-transaction-format.html#multi-signing) | 10 drops × (1 + Number of Signatures Provided) |
+| [EscrowFinish Transaction with Fulfillment](reference-transaction-format.html#escrowfinish) | 42 drops + (Fulfillment size in bytes divided by 16) |
```

Comment:
Mixing '+' and 'divided by' makes this formula a bit hard for me to read. Perhaps:

> | [EscrowFinish Transaction with Fulfillment](reference-transaction-format.html#escrowfinish) | 42 drops + (Fulfillment size (bytes) ÷ 16) |